### PR TITLE
Update free-programming-books-langs.md

### PR DESCRIPTION
**Pull request by hriprsd**
Added jenkins ebooks pdf links

## What does this PR do?
Add resource(s) 

### Why is this valuable (or not)?
Solves #10031 

### How do we know it's really free?
I have used it before and also, its from trusted free use sources.

### For book lists, is it a book? For course lists, is it a course? etc.
These are books that guide reader through Jenkins learning.

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1389,8 +1389,11 @@ Books on general-purpose programming that don't focus on a specific language are
 
 ### Jenkins
 
+* [Jenkins Essentials](https://raw.githubusercontent.com/rootusercop/Free-DevOps-Books-1/6dd8ee5bef93d2a1489a3761515ecbaf16c118cc/book/Jenkins%20Essentials.pdf) (PDF)
+* [Jenkins Starter Guide](https://bugfender.com/wp-content/themes/bugfender-wordpress-theme/assets/docs/Jenkins-Starter-Guide-Ebook.pdf) (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1393]:
```suggestion
* [Jenkins Starter Guide Ebook](https://bugfender.com/wp-content/themes/bugfender-wordpress-theme/assets/docs/Jenkins-Starter-Guide-Ebook.pdf) (PDF)
```
Don't invent titles! Use the title given by the resource.

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1389,8 +1389,11 @@ Books on general-purpose programming that don't focus on a specific language are
 
 ### Jenkins
 
+* [Jenkins Essentials](https://raw.githubusercontent.com/rootusercop/Free-DevOps-Books-1/6dd8ee5bef93d2a1489a3761515ecbaf16c118cc/book/Jenkins%20Essentials.pdf) (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1392]: This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines. Remove it

**Review by eshellman — CHANGES_REQUESTED** [review]

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -1389,8 +1389,11 @@ Books on general-purpose programming that don't focus on a specific language are
...
+* [Jenkins Starter Guide](https://bugfender.com/wp-content/themes/bugfender-wordpress-theme/assets/docs/Jenkins-Starter-Guide-Ebook.pdf) (PDF)
 * [Jenkins: The Definitive Guide](http://www.bogotobogo.com/DevOps/Jenkins/images/Intro_install/jenkins-the-definitive-guide.pdf) (PDF)
 * [Jenkins User Handbook](https://www.jenkins.io/user-handbook.pdf) (PDF)
+* [Learning Jenkins](https://riptutorial.com/Download/jenkins.pdf) (PDF)
```
[the remark this answers is not in the document]
Comment (hriprsd) [reply books/free-programming-books-langs.md]: Thanks for the review. Fixed review comments.

File: books/free-programming-books-langs.md [hunk 4]
```diff
@@ -1389,8 +1389,11 @@ Books on general-purpose programming that don't focus on a specific language are
 
 ### Jenkins
 
+* [Jenkins 2: Up and Running](https://theswissbay.ch/pdf/Books/Computer%20science/jenkins2_upandrunning.pdf) (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1392]: This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines. remove it.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**